corsheaders/middleware.py

```diff
@@ -103,7 +103,7 @@ def process_response(self, request, response):
         Add the respective CORS headers
         """
         origin = request.META.get('HTTP_ORIGIN')
-        if request._cors_enabled and origin:
+        if getattr(request, '_cors_enabled', False) and origin:
```

Comment:
Thanks. Indeed backwards compatibility trumps consistency. I've updated the code to that effect now.